File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/Java/C-Sheet #2-Loop/P.java
import java.util.Scanner;

public class P {
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);
  int num ;
  num = input.nextInt();
  for (int i = 1;i <= num;i++){
    for (int x = num ;x >= i;x--){
      System.out.print("*");
    }
    System.out.println();
  }
  input.close();
  }
}

File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/C/C-Sheet #2-Loop/P.c
#include <stdio.h>

int main() {
  int num ;
  scanf("%d", &num);
  for (int i = 1;i <= num;i++){
    for (int x = num ;x >= i;x--){
      printf("*");
    }
      printf("\n");
  }

  return 0;
}
